src/bindings.d.ts

```diff
@@ -1,18 +1,20 @@
-import { CID } from '@web3-storage/gateway-lib/handlers'
 import { Environment as RateLimiterEnvironment } from './handlers/rate-limiter.types.ts'
 import { Environment as CarBlockEnvironment } from './handlers/car-block.types.ts'
+import { Environment as EgressTrackerEnvironment } from './handlers/egress-tracker.types.ts'
+import { UnknownLink } from 'multiformats'
 
-export interface Environment extends CarBlockEnvironment, RateLimiterEnvironment {
+export interface Environment extends CarBlockEnvironment, RateLimiterEnvironment, EgressTrackerEnvironment {
   VERSION: string
   CONTENT_CLAIMS_SERVICE_URL?: string
+  ACCOUNTING_SERVICE_URL: string
 }
 
 export interface AccountingService {
-  record: (cid: CID, options: GetCIDRequestConfig) => Promise<void>
+  record: (resource: UnknownLink, bytes: number, servedAt: string) => Promise<void>
```

Comment:
**thought:** I think we need the Space here, not the CID: the CID doesn't correspond to (necessarily) exactly one Space, and therefore customer. Luckily, we'll have that in the context with my PR!

Reply:
That's right. I will add that one in the PR that integrates with `ucanto` lib.